### ISSUE TITLE
Any instructions to get started?

### ISSUE THREAD

**Issue by madcoda** [g0vhk-io/g0vhk_legco_web#2]
I am new to Python and Django, can you provide instructions to get the code running in the dev environment? I would like to contribute to the search engine (if I can handle the language)

**Comment by 3dd13** [comment]
added a readme, can have a look at my branch before it is merged
https://github.com/g0vhk-io/g0vhk_legco_web/pull/1

**Comment by 3dd13** [comment]
btw, I am new here as well ...

just had a glance at the source code
I think the followings are related to retrieving and parsing the data

```
gov_track_hk_web/xmls/download.py
gov_track_hk_web/api/management/commands/fetch_consultations.py
gov_track_hk_web/legco/management/commands
```

**Comment by madcoda** [comment]
Thanks for the pointer, much appreciated

**Comment by howawong** [comment]
not yet, I need to write README.md.

**Comment by howawong** [comment]
@madcoda  I had merged #1 , see if it is better now.
